packages/relay-runtime/store/__tests__/__generated__/RelayModernEnvironmentExecuteSubscriptionWithStreamTestCommentCreateSubscription.graphql.js

```diff
@@ -176,9 +176,7 @@ return {
   "params": {
     "cacheID": "c13dedba1d0c8b17a192b735d141de99",
     "id": null,
-    "metadata": {
```

Comment:
Hmm, why this has changed?

Reply:
`scripts/compile-tests.sh` was broken on `main` for me.

I added `"enable_provided_variables": {"kind": "enabled"}` to fix, and running it generated this change.

(maybe I was doing something wrong?)